demo/regression/README.md

```diff
@@ -64,24 +66,34 @@ module = hub.Module(name="bert_cased_L-12_H-768_A-12")
 
 ### Step2: 准备数据集并使用RegressionReader读取数据
 ```python
-dataset = hub.dataset.GLUE("STS-B")
-reader = hub.reader.RegressionReader(
-    dataset=dataset,
-    vocab_path=module.get_vocab_path(),
-    max_seq_len=args.max_seq_len)
+tokenizer = hub.BertTokenizer(vocab_file=module.get_vocab_path())
```

Comment:
这里应该把 参数 tokenize_chinese_chars置为false，默认为true，同理检查下demo

Reply:
# This was added on November 1st, 2018 for the multilingual and Chinese
# models. This is also applied to the English models now, but it doesn't
# matter since the English models were not trained on any Chinese data
# and generally don't have any Chinese data in them (there are Chinese
# characters in the vocabulary because Wikipedia does have some Chinese
# words in the English Wikipedia.).

只有日语要置False